html/forms/form-validation/full-example.html

```diff
@@ -36,16 +36,14 @@
 
 <body>
   <form>
-    <p>
-      <fieldset>
-        <legend>Do you have a driver's license?<span aria-label="required">*</span></legend>
-        <!-- While only one radio button in a same-named group can be selected at a time,
-             and therefore only one radio button in a same-named group having the "required"
-             attribute suffices in making a selection a requirement -->
-        <input type="radio" required name="driver" id="r1" value="yes"><label for="r1">Yes</label>
-        <input type="radio" required name="driver" id="r2" value="no"><label for="r2">No</label>
-      </fieldset>
-    </p>
+    <fieldset>
+      <legend>Do you have a driver's license?<span aria-label="required">*</span></legend>
+      <!-- While only one radio button in a same-named group can be selected at a time,
+           and therefore only one radio button in a same-named group having the "required"
+           attribute suffices in making a selection a requirement -->
+      <input type="radio" required name="driver" id="r1" value="yes"><label for="r1">Yes</label>
+      <input type="radio" required name="driver" id="r2" value="no"><label for="r2">No</label>
+    </fieldset>
```

Comment:
Why the alignment here is different than the one on mdn/content?

Reply:
These bugs were preventing Prettier from formatting. So I pulled the fixes first thinking I would run Prettier later.

To make the fix whole let us commit the formatted file here itself:https://github.com/mdn/learning-area/pull/695/commits/6af44c9184c6939c4428bb8a226a171f34feb1b0